src/test/java/org/folio/service/processing/reader/SourceReaderBuilderTest.java

```diff
@@ -36,6 +42,7 @@ public void buildShouldReturnMarcJsonReader() {
     //when
     SourceReader reader = SourceReaderBuilder.build(new File(SOURCE_JSON_PATH), marcJobProfile);
     //then
+    Assert.assertNotNull(reader);
```

Comment:
Reader can not be null in any case, because it is always previously created

Reply:
The tests are updated, thanks